includes/polling/ports/ciscon3550.inc.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * ciscon3550.inc.php
```

Comment:
I think this file is getting called twice by the poller because of this on line 320 of ports.inc.php
```php
if (file_exists($os_file)) {
    require $os_file;
}
```